kinder/pkg/test/workflow/workflow.go

```diff
@@ -216,7 +217,7 @@ func (w *Workflow) expandImports(file string) error {
 				w.Vars[k] = v
 				continue
 			}
-			log.Infof("var %s in workflow file %s is shadowed by var %[1]s in parent workflow file %[3]s", k, path, file)
+			log.Warningf("var %s in workflow file %s is shadowed by var %[1]s in parent workflow file %[3]s", k, path, file)
```

Comment:
[1] I think we should downgrade this as a debug message because we are heavily relying on variable shadowing, and this message make s the output of verify-workflows really verbose without adding much value. wdyt?

Reply:
ok, will revert to infof